airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/manifest_only_connectors.py

```diff
@@ -46,3 +54,19 @@ def get_test_steps(context: ConnectorTestContext) -> STEP_TREE:
             )
         ],
     ]
+
+
+class ManifestOnlyConnectorUnitTests(PytestStep):
+    """A step to run unit tests for a manifest-only connector"""
+
+    title = "Manifest-only unit tests"
+    test_directory_name = "unit_tests"
+
+    # No coverage reporting needed since we're just testing components.py
```

Comment:
Question: should we actually be maintaining coverage expectations? I feel like it's not quite as relevant as the days of Python sources yore, but I'm happy to be wrong if we still want to guarantee a certain threshold for certified connectors.

Reply:
Nope. Honestly if we do our job of abstractions in the CDK correctly (eventually) custom components should be sparse enough that we dont get much from mandating unit test coverage